src/ui/server/test-mocks/mock-buyer.ts

```diff
@@ -4,7 +4,7 @@ dotenv.config();
 
 const mockBuyer = {
   companyOrOrganisationName: 'Test name',
-  address: 'Test address',
+  address: 'Test \r\n address',
```

Comment:
Not a blocker - a more complete/populated mock address e.g 3 lines country etc would be great

Reply:
will do in next PR